### PR TITLE
[13.0][MIG] mgmtsystem_claim

### PR DESCRIPTION
**Pull request by flachica**
Depends on:
- [ ] #311

### CODE REVIEW

**Comment by flachica** [comment]
This [fail](https://travis-ci.org/github/OCA/management-system/jobs/665907128#L775) is because need wait to this [PR](https://github.com/OCA/crm/pull/311) be merged

**Comment by FernandoRomera** [comment]
@flachica, could you rebase this PR to continue it?.

**Comment by flachica** [comment]
> @flachica, could you rebase this PR to continue it?.

Forced push

**Comment by FernandoRomera** [comment]
Have you rebased over current 13.0 branch?

**Comment by flachica** [comment]
> Have you rebased over current 13.0 branch?

Yes, but as you can see [here](https://travis-ci.org/github/OCA/management-system/jobs/700977318#L789), this module depends of OCA/crm/crm_claim. crm_claim is migrated yet but travis does not pulled it

**Comment by flachica** [comment]
> Have you rebased over current 13.0 branch?

I included crm on oca_dependecies and test was passed

File: mgmtsystem_claim/views/mgmtsystem_claim.xml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+        <!-- Claim Search view -->
+        <record id="view_mgmtsystem_claims_filter" model="ir.ui.view">
+            <field name="name">Management System Claims Search</field>
+            <field name="model">mgmtsystem.claim</field>
+            <field name="arch" type="xml">
+                <search string="Search Claims">
+
+                    <filter icon="terp-check" string="Current" name="current"
+                            domain="[('stage','in',(%(mgmtsystem_claim.stage_draft)d, %(mgmtsystem_claim.stage_open)d))]"
```
Comment (FernandoRomera) [inline mgmtsystem_claim/views/mgmtsystem_claim.xml]: Replace domain="[('stage','in', by domain="[('stage_id','in',

File: mgmtsystem_claim/views/mgmtsystem_claim.xml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                            separator="1" help="Draft and Open Claims"/>
+
+                    <filter icon="terp-gtk-media-pause"
+                            string="Pending"
+                            name="pending"
+                            domain="[('stage','=', %(mgmtsystem_claim.stage_pending)d)]"
```
Comment (FernandoRomera) [inline mgmtsystem_claim/views/mgmtsystem_claim.xml]: Replace domain="[('stage','=', by domain="[('stage_id','=',

**Review by FernandoRomera — CHANGES_REQUESTED** [review]

**Review by FernandoRomera — APPROVED** [review]
Functional review, ok.

**Review by dreispt — APPROVED** [review]

**Pull request merged**